hack/make-rules/test.sh

```diff
@@ -186,7 +186,7 @@ junitFilenamePrefix() {
   # barely fits there and in coverage mode test names are
   # appended to generated file names, easily exceeding
   # 255 chars in length. So let's just sha1sum it.
```

Comment:
s/sha1sum/shasum with `-a 1` option.

One question is that I recalled this works well about a month ago, why it failed now?